cmd/init.go

```diff
@@ -16,25 +16,55 @@ func Init(repoName string, tp node.Type) *cobra.Command {
 		panic("parent command must specify a persistent flag name for repository path")
 	}
 
-	const cfgName = "config"
+	const (
+		nodeConfig  = "node.config"
+		genesis     = "headers.genesis-hash"
+		trustedPeer = "headers.trusted-peer"
+		coreRemote  = "core.remote"
+	)
 	cmd := &cobra.Command{
-		Use:  "init",
-		Args: cobra.NoArgs,
+		Use:   "init",
+		Short: "Initialization for Celestia Node. Passed flags have persisted effect.",
```

Comment:
what does this mean?

Reply:
It means that the options that alter config are getting saved with config on init.